enterprise/frontend/src/embedding-sdk/cli/snippets/analytics-css-snippet.ts

```diff
@@ -46,4 +51,15 @@ export const ANALYTICS_CSS_SNIPPET = `
   outline: 1px solid #509EE3;
   border-radius: 2px;
 }
+
+.analytics-auth-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+
+  color: var(--mb-color-text-primary);
```

Comment:
(TODO: replace this by adding the `light/dark` theme CSS classes to body, and determine the color from there. this is temporary)